app/api/v2/identity/utils.rb

```diff
@@ -24,16 +24,16 @@ def language
       end
 
       def verify_captcha!(user:, response:, error_statuses: [400, 422])
-        return if Barong::CaptchaPolicy.config.disabled
+        return if Barong::App.config.barong_captcha == 'disabled'
 
         if response.blank?
           error!({ errors: ['identity.captcha.required'] }, error_statuses.first)
         end
 
-        if Barong::CaptchaPolicy.config.re_captcha
+        if Barong::App.config.barong_captcha == 're_captcha'
```

Comment:
Spell it recapcha like we do on frontend

Reply:
now its 'recaptcha'